sdk/data/azcosmos/ci.yml

```diff
@@ -24,4 +24,57 @@ stages:
 - template: /eng/pipelines/templates/jobs/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: 'data/azcosmos'
-    RunTests: true
\ No newline at end of file
+    RunTests: true
+- stage: Emulator
+  displayName: 'Cosmos Emulator'
+  jobs:
+  - job: DownloadAndRunCosmosEmulator
+    displayName: Download and run Cosmos Emulator
+
+    strategy:
+      matrix:
+        Windows_Go116:
+          pool.name: azsdk-pool-mms-win-2019-general
```

Comment:
@benbp at some point we should normalize the name of the pool and image in the go repo. We have common templates and scripts that expect this to be named OSVmImage for example.